Packs/WALLIXBastion/ReleaseNotes/3_0_0.md

```diff
@@ -0,0 +1,86 @@
+
+#### Integrations
+
+##### WALLIX Bastion
+- Updated the Docker image to: *demisto/python3:3.11.10.111526*.
+
+
+- The integration now returns the ID of created objects on the Bastion
+
+- Added support for nullable arguments
+
+- Added suggestions for enum arguments (i.e having predefined values) and booleans
+
+- fix support for empty lists arguments
+
+- Added new commands:
+    - wab-add-connection-policy
+    - wab-add-mapping-in-domain
+    - wab-add-password-change-policy
+    - wab-add-restriction-to-usergroup
+    - wab-add-timeframe
+    - wab-add-timeframe-period
+    - wab-change-password-or-ssh-key-of-account
+    - wab-delete-connection-policy
+    - wab-delete-mapping-of-domain
+    - wab-delete-password-change-policy
+    - wab-delete-restriction-from-targetgroup
+    - wab-delete-restriction-from-usergroup
+    - wab-delete-scan
+    - wab-delete-timeframe
+    - wab-edit-connection-policy
+    - wab-edit-mapping-of-domain
+    - wab-edit-mappings-of-domain
+    - wab-edit-password-change-policy
+    - wab-edit-restriction-from-targetgroup
+    - wab-edit-restriction-from-usergroup
+    - wab-edit-scan
+    - wab-edit-timeframe
+    - wab-edit-user
+    - wab-get-cluster
+    - wab-get-clusters
+    - wab-get-connection-policies
+    - wab-get-connection-policy
+    - wab-get-external-authentication-group-mappings
+    - wab-get-local-domain-data-for-application
+    - wab-get-local-domain-of-device
+    - wab-get-local-domains-data-for-application
+    - wab-get-local-domains-of-device
+    - wab-get-mapping-of-domain
+    - wab-get-mappings-of-domain
+    - wab-get-target-group-restriction
+    - wab-get-user-group-restriction
+    - wab-get-password-change-policies
+    - wab-get-password-change-policy
+    - wab-get-passwordrights
+    - wab-get-passwordrights-user-name
+    - wab-get-version
+    - wab-get-target-group-restrictions
+    - wab-get-timeframe
+    - wab-get-timeframes
+    - wab-get-user-group-restrictions
+
+- wab-add-service-in-device:
+    - set service_post_subprotocols as non required 
+    - add service_post_seamless_connection option for RAWTCPIP
+
+- wab-edit-service-of-device: 
+    - add optional parameters service_post_subprotocols
+    - add service_put_seamless_connection option for RAWTCPIP
+
+- wab-edit-application: add following optional parameters:
+    - add optional parameters application_put_global_domains
+    
+- **Breaking changes:** removed deprecated command wab-get-metadata-of-one-or-multiple-sessions
+
+- **Breaking changes:** removed deprecated arguments in:
+    - wab-edit-application: application_put__meters
+
+- **Breaking changes:** removed deprecated outputs in:
```

Comment:
please spilt so each remove in a command will be in a separate line
e.g., removed the XXX outputs in the YYY coammand.

Reply:
done